tools/validate/src/dependencies/fixDependencyIssues.ts

```diff
@@ -22,15 +22,14 @@ export async function fixDependencies(
     stdio: 'inherit',
     cwd: `packages/${pkg}`,
   };
-  // Using yarn 1.19.0 https://stackoverflow.com/questions/62254089/expected-workspace-package-to-exist-for-sane
 
   // Install any missing dependencies
   if (missingDependencies.length > 0) {
     verbose &&
       console.log('Installing missing dependencies...', missingDependencies);
     spawnSync(
       'npx',
-      ['yarn@1.19.0', 'add', ...missingDependencies],
+      ['pnpm@9.15.0', 'add', ...missingDependencies],
```

Comment:
I don't know if we need to specify the version here. We have to do that with `yarn` since there was a bug in `>1.20.0`